src/vivarium/framework/engine.py

```diff
@@ -207,6 +207,10 @@ def __init__(
     def name(self) -> str:
         return self._name
 
+    @property
+    def results(self) -> Dict[str, Any]:
+        return self._results.metrics
```

Comment:
Since we still have the "metrics" pipeline (though we may eventually get rid of it), would it be better to do this?
```suggestion
        return self._values.get_value("metrics")(self.get_population().index)
```

Reply:
I think this needs some re-work. The problem is that down in the `report` method, it calls the pipeline as well. And so we may be modifying the pipelines more than we intend if we start calling it from multiple locations.

I'm actually getting a breaking test with the way the metrics pipeline is set up since it's a acting as a counter every time it's called. That's probably just a biproduct of a silly test fixture but I think still highlights the potential issue?

Reply:
I'm confused what the purpose of the `metrics` property is then. We have a `ResultsManagwer._metrics` attribute (defaultdict[str, pd.DataFrame] of measure-specific results), a `ResultsManager.metrics` property which is just `self._metrics.copy()`, and then the `metrics` pipeline which is an empty dict and has a value modifer that just updates it w/ the metrics proeprty (`metrics.update(self.metrics)`).